apps/microservices-orders/orders-service/Makefile

```diff
@@ -0,0 +1,28 @@
+.PHONY: *
+
+HELP_TAB_WIDTH = 25
+
+.DEFAULT_GOAL := help
+
+SHELL=/bin/bash -o pipefail
+
+check-dependency = $(if $(shell command -v $(1)),,$(error Make sure $(1) is installed))
+
+clean:
+	@./gradlew clean
+
+test:
+	@./gradlew test
+
+build:
+	@./gradlew shadowJar
```

Comment:
shadowJar is redundand. boot plugin can do it `bootJar - Assembles an executable jar archive containing the main classes and their dependencies.`